src/fund_my_watcard/logger.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import colorlog
+from .config_file import CONFIG_FILE_PATH
+from .util import PRINT_PREFIX
+
+# add new level - SUCCESS
+SUCCESS = logging.ERROR - 1
+logging.addLevelName(SUCCESS, "SUCCESS")
+
+
+def success(self, msg, lvl=SUCCESS, *args, **kws):
+    self.log(lvl, msg, *args, **kws)
+
+
+logging.Logger.success = success
+
+
+def init_logger():
+    # create logger
+    logger = logging.getLogger("logger")
+    logger.setLevel(logging.INFO)
+
+    # create file handler and set level to INFO
+    logfile = CONFIG_FILE_PATH.replace("config", "log")
```

Comment:
What if a user's name is config? E.g., `/home/config/.watcard_config`. I know it is rare, but we need to consider this case.

Reply:
It would be better to use a common path for these files. All generated files will in this path.